Cloud Application Manager/Datacenter Edition/camdce-upgrading.md

```diff
@@ -1,12 +1,12 @@
 {{{
 "title": "Upgrading the Appliance",
-"date": "09-01-2016",
+"date": "05-24-2019",
```

Comment:
Please, change title, author and keywords in all files

Reply:
done